deepchem/feat/roberta_tokenizer.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, **kwargs):
     super().__init__(**kwargs)
     return
 
-  def _featurize(self, sequence: str, **kwargs) -> List[List[int]]:
+  def _featurize(self, datapoint: str, **kwargs) -> List[List[int]]:
     """Calculate encoding using HuggingFace's RobertaTokenizerFast
 
     Parameters
```

Comment:
The parameter name needs to be changed here as well I think

Reply:
done! thought so as well to get mypy to pass :)